os/net/routing/rpl-classic/rpl-dag.c

```diff
@@ -543,17 +542,17 @@ int
 rpl_set_default_route(rpl_instance_t *instance, uip_ipaddr_t *from)
 {
   if(instance->def_route != NULL) {
-    PRINTF("RPL: Removing default route through ");
-    PRINT6ADDR(&instance->def_route->ipaddr);
-    PRINTF("\n");
+    LOG_DBG_("Removing default route through ");
```

Comment:
this should start a new line, i.e., no ending `_`

Reply:
It immediately logs the ip6 address for the route and prints newline afterwards. Is there a different way i should be calling this to log the address?

Reply:
no, talking about the line above the address logging. Should be
```
LOG_DBG("Removing default route through ");
```

Reply:
Oh my confusion! For some reason i though the `LOG_DBG()` automatically added newline. Fixing.